ckanext/dia/harvester/dcat.py

```diff
@@ -76,15 +79,25 @@ def _fetch_license_id(self, license_url):
         licenses = license_list({'model': model}, {})
         try:
             resp = requests.get(license_url)
-        except Exception:
-            log.exception(
-                "Failed to get on license url: {}".format(license_url))
+        except (requests.exceptions.InvalidSchema,
+                requests.exceptions.InvalidURL,
+                requests.exceptions.MissingSchema):
+            self._save_object_error(
+                f'License URL is invalid.',
+                harvest_object)
```

Comment:
Ah, I see `_save_object_error` is how we present error messages to the harvest UI, nice.

Reply:
Yeah its a nice way to provide the feedback to users rather than be hidden in the logs.